### DISCUSSION TITLE
Yandex ads

### DISCUSSION THREAD

**Discussion by Jokopentil** [uBlockOrigin/uBOL-home#130] [Q&A]
how to block video ads from Yandex that have a star logo
is there a filter?
https://yandex.eu/video/touch/preview/15333762920045686363
![IMG_20240512_130714.jpg](https://github.com/uBlockOrigin/uBOL-home/assets/114223791/61ac6cd4-14b5-4c0c-b4e4-ae80b17e56f2)

**Comment by dportvine** [comment]
https://github.com/easylist/easylist/discussions/19102
